src/tests/functional/plugin/shared/src/main.cpp

```diff
@@ -32,6 +34,10 @@ int main(int argc, char *argv[]) {
                 throw std::runtime_error("Incorrect value of \"--save_report_timeout\" argument");
             }
             ov::test::utils::OpSummary::setSaveReportTimeout(timeout);
+        } else if (std::string(argv[i]).find("--device_suffix") != std::string::npos) {
+            std::string deviceSuffix = std::string(argv[i]).substr(std::string("--device_suffix").length() + 1);
+            // TODO: uncomment after plugin impl merge
```

Comment:
Do we have a plan for this?

Reply:
of course.
I will uncomment it after merge to other plugins